packages/shortcuts-extension/src/components/ShortcutInput.tsx

```diff
@@ -162,6 +162,10 @@ export class ShortcutInput extends React.Component<
         keys: keys,
         currentChain: currentChain
       });
+    } else if (event.key === 'Escape') {
+      const focusedElement = document.activeElement;
+      const parentRow = focusedElement?.closest('.jp-Shortcuts-Row');
+      (parentRow as HTMLDivElement).focus();
```

Comment:
This will error out if active element is outside of the of table. It should not usually happen but any casting with `as` is unsafe because it is an implicit non-null assertion.